ui.content/src/main/content/jcr_root/var/.content-cloud.xml

```diff
@@ -0,0 +1,5 @@
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0"
```

Comment:
I couldn't find any other solution for the `cloud` profile than to add the empty `var` folder.